terraform.md

```diff
@@ -50,14 +50,41 @@ variable "web_cidrs" {
 }
 ```
 
-### Defaults
-
+### Value Assignment
+Values can be assigned to variables in Environment 'vars' files, variable block 'defaults' or 'locals' blocks. Ensure the most appropriate method is used for each definition to aid readability and avoid duplication.  
+Some guidance:
+#### Defaults
 If it makes sense to do so then default values should be provided. Defaults are definitely preferable to duplicated values between environments
+#### Environment vars
+Set values here only for variables that will change per environment.
+#### Locals Blocks
```

Comment:
I'd rename this to `Local vars` in keeping with `Environment vars`

Reply:
Done.